modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -13,22 +13,22 @@ You must specify a storage backend in the image set configuration file. This sto
 
 [IMPORTANT]
 ====
-Do not delete or modify the metadata that is generated by the oc-mirror plugin. You must use the same storage backend every time you run the oc-mirror plugin for the same mirror registry.
+Do not delete or modify the metadata that is generated by the oc-mirror plugin v2. You must use the same storage backend every time you run the oc-mirror plugin for the same mirror registry.
```

Comment:
This addition of "v2" is also showing up in the v1 version of the oc-mirror docs. Perhaps consider leaving it off - I assume this is true of both versions of oc-mirror?